src/script.js

```diff
@@ -117,46 +137,66 @@ var gianniAccordion = (function(){
       this.transitionendevent = this.transitionendEventName();
       this.oneAtATime = true;
       this.selectedClass = "selected";
+      this.expandedClass = "expanded";
+      this.collapsedClass = "collapsed";
       this.trigger = "[data-accordion-element-trigger]";
       this.content = "[data-accordion-element-content]";
+      this.collapsedHeight = '0px';
+      this.defaultElements = ['a', 'button', 'input'];
+      this.openAtLandingIndex = -1
     }
 
-    setCustomData(data){
+    setCustomData(data) {
       var keys = Object.keys(data);
 
-      for(var i=0; i<keys.length; i++){
-        this[ keys[i] ] = data[ keys[i] ];
+      for (var i = 0; i < keys.length; i++) {
+        this[keys[i]] = data[keys[i]];
       }
     }
 
-    fire(eventName, el){
+    setLandinData(el) {
```

Comment:
setQualcosa**Data** è usato per settare dati (variabili), non per modificare il dom. 
Chiamalo collapseAtLanding e aggiungici anche 
_element.content.style.height = this.collapsedHeight_
che attualmente è nel constructor subito prima della chiamata a questa funzione